src/App.js

```diff
@@ -1,22 +1,40 @@
 import React from 'react';
 import './App.css';
 import Settings from './routes/settings';
-import SurveyView from './routes/survey';
-import Search from './routes/search';
 import ViewSpace from './routes/view-space/the-think-tank';
-import Homebar from './components/homebar';
-
+import Search from './routes/search';
+import {
+  Survey,
+  Homebar
+} from './components';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
+  Redirect
 } from "react-router-dom";
+import LandingPage from './routes/landingPage/LandingPage';
+import HomePage from './routes/homePage';
 
 export default class App extends React.Component {
+  constructor(props) {
```

Comment:
Temporary State - was trying to get routing to work between login/home screen. Will need help fixing this later.